ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -403,44 +404,29 @@ deriving instance Eq       (Core.TxOut era) => Eq       (TxOutWrapper era)
 deriving instance NoThunks (Core.TxOut era) => NoThunks (TxOutWrapper era)
 deriving instance Show     (Core.TxOut era) => Show     (TxOutWrapper era)
 
+class CanTranslateTxOut era where
+  translateTxOut :: Core.TxOut (PreviousEra era) -> Maybe (Core.TxOut era)
```

Comment:
If we have this class, it seems like we won't need the type-dependent translation contexts in ledger

Reply:
Yeah, we don't need it _for now_